src/status_im/utils/prices.cljs

```diff
@@ -13,19 +13,24 @@
 (def api-url "https://min-api.cryptocompare.com/data")
 (def status-identifier "extraParams=Status.im")
 
+(defn ->url-param-syms [syms]
+  (if (coll? syms)
+    ((comp (partial clojure.string/join ",") (partial map name)) syms)
+    syms))
+
 (defn- gen-price-url [fsyms tsyms]
-  (str api-url "/pricemultifull?fsyms=" fsyms "&tsyms=" tsyms "&" status-identifier))
+  (str api-url "/pricemultifull?fsyms=" (->url-param-syms fsyms) "&tsyms=" (->url-param-syms tsyms) "&" status-identifier))
 
-(defn- format-price-resp [from to resp]
-  (let [raw (:RAW (types/json->clj resp))
-        entry (get-in raw [(keyword from) (keyword to)])]
-    {:from         from
-     :to           to
-     :price        (:PRICE entry)
-     :last-day     (:OPEN24HOUR entry)}))
+(defn- format-price-resp [resp]
```

Comment:
This also looks like a good candidate for a test, you can test it through `get-prices` if you want to be strict about not testing private fn, or make it public, I would be ok with both approaches, unless someone disagree.